### ISSUE TITLE
Pick fails at purchasing lockpicks

### ISSUE THREAD

**Issue by jandersson** [rpherbig/dr-scripts#470]
The hardcoded lockpick prices do not match the offered prices (due to charisma or price change or etc...)
Suggestion is to use the hardcoded prices as an upper bound to determine how much money to pull from the bank, but retrieve the offered value from the game instead.

```
[go2: travel time: 0:00:16]
--- Lich: go2 has exited.
[pick]>buy stout lockpick

Ragge sighs.  "Despite the rarity of this lockpick, I'm prepared to offer it to you for 237 kronars."
>
[pick]>offer 250
An offer of 250?  Are you SURE?  (OFFER it again within the next 20 seconds if you are.)
>
[pick]>put my lockpick on my lockpick ring
>
[pick]>retreat
What were you referring to?
>
You are already as far away as you can get!
>
--- Lich: go2 active.
[go2: ETA: 0:00:01 (9 rooms to move through)]
[go2]>out
```

**Comment by jandersson** [comment]
Alternatively, you could just offer twice if "Are you sure" is seen, but that seems wasteful

**Comment by rpherbig** [comment]
Thanks for the PR!

**Comment by rpherbig** [comment]
Closed by #478
